### PR TITLE
Implementing GasStation:GasStation() function

### PR DESCRIPTION
**Pull request by danielhejduk**
This PR does:

Implementing `GasStation:GasStation()` in `LEGO1/lego/legoomni/src/gasstation/gasstation.cpp`

### CODE REVIEW

**Comment by ItzSwirlz** [comment]
o/ welcome! The format for naming variables is m_unk0x(foo)

**Comment by danielhejduk** [comment]
@ItzSwirlz thank you so much!

**Comment by danielhejduk** [comment]
Sorry maintainers, I forgor :skull: the `this` keyword please start the github actions rewiew again.

**Comment by ItzSwirlz** [comment]
GH Actions always runs every commit - you also need to update lego/legoomni/include/gasstation.h

**Comment by danielhejduk** [comment]
Sorry, I am new for this

**Comment by danielhejduk** [comment]
I fixed the error everything is passing

File: LEGO1/lego/legoomni/include/gasstation.h [hunk 1]
```diff
@@ -26,6 +28,20 @@ class GasStation : public LegoWorld {
 	{
 		return !strcmp(p_name, GasStation::ClassName()) || LegoWorld::IsA(p_name);
 	}
+
+private:
+	undefined2 m_unk0xf8;
```
Comment (foxtacles) [inline LEGO1/lego/legoomni/include/gasstation.h:33]: Still needs the offset annotations here, but then it should be good to merge

**Pull request merged**